Jamfile

```diff
@@ -244,7 +244,6 @@ rule warnings ( properties * )
 
 # enable these warnings again, once the other ones are dealt with
 		result += <cflags>-Wno-weak-vtables ;
-		result += <cflags>-Wno-sign-conversion ;
```

Comment:
:) yes it is, C4365 and it is activated with `/Wall`, I have a pending review in `Jamfile` and I will check this (and others flags).